README.md

```diff
@@ -1,15 +1,3 @@
 ```
-░░░░█─────────────█──▀──
-░░░░▓█───────▄▄▀▀█──────
-░░░░▒░█────▄█▒░░▄░█─────
-░░░░░░░▀▄─▄▀▒▀▀▀▄▄▀─────
-░░░░░░░░░█▒░░░░▄▀───────
-▒▒▒░░░░▄▀▒░░░░▄▀────────
-▓▓▓▓▒░█▒░░░░░█▄─────────
-█████▀▒░░░░░█░▀▄────────
-█████▒▒░░░▒█░░░▀▄───────
-███▓▓▒▒▒▀▀▀█▄░░░░█──────
-▓██▓▒▒▒▒▒▒▒▒▒█░░░░█─────
-▓▓█▓▒▒▒▒▒▒▓▒▒█░░░░░█────
-░▒▒▀▀▄▄▄▄█▄▄▀░░░░░░░█───
+8===D
```

Comment:
Should I add a `~` to look like semen? The original seems to have semen (or urine). 🤷‍♀️